pkg/process/procutil/resources/test_procfs/proc/1/cwd

```diff
@@ -0,0 +1 @@
+../../test_cwd
```

Comment:
If you view the file you can see it's a symlink
![image](https://user-images.githubusercontent.com/1155971/182950397-102b929a-0cd3-4b9e-abe3-5c254e5faed2.png)
https://github.com/DataDog/datadog-agent/blob/face96a/pkg/process/procutil/resources/test_procfs/proc/1/cwd